ui/raidboss/data/04-sb/raid/o7n.ts

```diff
@@ -71,6 +100,55 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.headMarker({ id: '001E' }),
       response: Responses.preyOn('info'),
     },
+    {
+      id: 'O7N Bomb Deployment',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '2762', source: 'Guardian', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2762', source: 'Wächter', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2762', source: 'Gardien', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2762', source: 'ガーディアン', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2762', source: '守护者', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2762', source: '가디언', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Stand in glowing bomb circle',
+        },
+      },
+    },
+    {
+      id: 'O7N Demon Simulation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '2752', source: 'Guardian', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2752', source: 'Wächter', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2752', source: 'Gardien', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2752', source: 'ガーディアン', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2752', source: '守护者', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2752', source: '가디언', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Activate 3 person towers',
```

Comment:
This is probably as short as we can make any shot call for this one. Maybe "get 3-person towers", but it's all the same.

Reply:
Yeah, I was not that happy with this either.  This was the least worst idea I could come up with.